.circleci/config.yml

```diff
@@ -396,6 +427,19 @@ jobs:
             ./scripts/deploy.sh -o ios -t soft -e production
       - notify-release
 
+  deploy-decliweb:
+    docker:
+      - image: europ-west1-docker.pkg.dev/passculture-infra-prod/pass-culture-tools/node-gcp:12
+    working_directory: ~/pass-culture
+    steps:
+      - authenticate_gcp:
+          gcp_key_name: GCP_METIER_KEY
+      - build:
+          node_env: testing
+      - push-to-bucket:
+          build_path: ~/pass-culture-app-native/build/
+          bucket_name: ${GCP_PROJECT}-testing-decliweb
```

Comment:
pour ces deux variable je pense que tu ne les as pas utilisées. Si tu comptes les utiliser il faut renseigner dans le job push-to-bucket  la commande à executer : 
`gsutil rsync -r << parameters.build_path >> gs://<< parameters.bucket_name >>` ca va remplacer parameters.build_path et parameters.bucket_name avec les valeurs des variables